frontend/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
We can't execute both entrypoint files when the pods start up.  
I would be okay with either of these options:
A) the frontend-legacy and frontend share the same entrypoint file
B) one script calls the other and then resumes its regular execution

Reply:
I just pushed an update to combine the logic into the legacy frontend's existing entrypoint script. We'll just need a `V2_APP_ROOT` var to point to the frontend/public directory.

If option B makes more sense for how we're building and deploying, just let me know and I can update it 👍